app/controllers/standup_meetings/skips_controller.rb

```diff
@@ -0,0 +1,11 @@
+class StandupMeetings::SkipsController < ApplicationController
+  def create
+    @standup_meeting = StandupMeeting.find(params[:standup_meeting_id])
```

Comment:
super small, but this method should have `includes(:standup_meeting_group)` exactly like the completions controller has.